blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseBlobIntegrationTest.java

```diff
@@ -799,7 +799,9 @@ public void testMetadata() throws InterruptedException, IOException {
          blob.getMetadata().getUserMetadata().put("Adrian", "wonderpuff");
          blob.getMetadata().getUserMetadata().put("Adrian", "powderpuff");
 
+         awaitConsistency();
          addBlobToContainer(container, blob);
+         awaitConsistency();
```

Comment:
If we change `removeBlob` will this again look strongly-consistent?  Also do we have any issues with overwriting blobs?